README.md

```diff
@@ -59,14 +59,14 @@ To build skeleton LLVM pass found in `skeleton` folder:
 $ cd llvm-pass-tutorial
 $ mkdir build
 $ cd build
-$ LLVM_DIR="$LLVM_HOME/share/llvm/cmake" cmake ..
+$ cmake ..
 $ make
 ```
 cmake needs to find its LLVM configurations which we provide using
 `$LLVM_DIR`. Now the easiest way to run the skeleton pass is using Clang which
 is the compiler front-end of the LLVM framework:
 ```bash
-$ clang-3.9 -Xclang -load -Xclang build/skeleton/libSkeletonPass.* something.c$
+$ $LLVM_HOME/bin/clang -Xclang -load -Xclang build/skeleton/libSkeletonPass.* something.c$
```

Comment:
`clang` and `llvm` are independent and this change can confuse newcomers into thinking that they have to install `clang` also from source in order to use `llvm`. Actually, if you are working only with analysis and optimization passes on LLVM IR, then you only need to work with `llvm`, preferably a debug version. A binary installation of `clang` should work as long as it is compatible with our `llvm` installation.

Reply:
Yes, I think clang-7.0 is more suitable. I add this because when I use `/usr/bin/clang` in my MacOS, it failed with "XXX" symbol not found. So at that time, I thought only the compiled-clang can use this command. It's my fault.